spec/tile-spec.yml

```diff
@@ -0,0 +1,245 @@
+swagger: '2.0'
+info:
+  title: Raster Foundry
+  description: An application to find, view, and analyze geospatial data at any scale
+  version: "0.1.0"
+
+host: backsplash.rasterfoundry.com
+
+schemes:
+  - http
+  - https
+
+basePath: /
+
+produces:
+  - application/json
+  - img/png
+consumes:
+  - application/json
+
+x-top-matter:
+  - title: Introduction
+    level: 1
+    content: |
+      The backsplash server serves tiles and statistical information about tile layers in the Raster Foundry application.
+  - title: Authentication
+    level: 2
+    content: |
+      You can authenticate with the backsplash tile server in one of three ways:
+        - mapToken query parameters: these are in the form of UUIDs included as query parameters. You can get one of these from a project in the Raster Foundry application.
+        - token  query parameter: This is a <a href="https://jwt.io/introduction/" target="_blank">JSON Web Token (JWT)</a> that you include in a query parameter called 'token'. It <i>does not</i> include the 'Bearer ' component of the token.
+        - authorization headers: This is a <a href="https://jwt.io/introduction/" target="_blank">JSON Web Token (JWT)</a>. You can create one with a refresh token created in the Raster Foundry application.
+
+x-resources:
+  - name: Mosaic
+    description: |
+      Single- or multi-band visualization of a project layer.
+  - name: Analysis
+    description: |
+      Visualization of the results of map algebraic computations
+  - name: Healthcheck
+    description: |
+      Assurance that the tile server is still alive
+
+tags:
+  - name: Mosaic
+    description: 'Interact with mosaics'
+  - name: Analysis
+    description: |
+      Visualization of the results of map algebraic computations
+  - name: Healthcheck
+    description: |
+      Assurance that the tile server is still alive
+
+paths:
+  /healthcheck/:
+    x-resource: Healthcheck
+    get:
+      summary: 'Confirm the tile server is available'
+      tags:
+        - Healthcheck
+      responses:
+        200:
+          description: 'OK'
+          schema:
+            type: object
+            properties:
+              message:
+                type: string
+              reason:
+                type: array
+                items:
+                  type: string
+
+  /tools/{analysisId}/{z}/{x}/{y}/:
+    x-resource: Anslysis
+    get:
+      produces:
+        - image/png
+      summary: Visualize an analysis (or node of an analysis) as a tile
+      tags:
+        - Analysis
+      parameters:
+        - $ref: '#/parameters/zoom'
```

Comment:
Should this match the value `{z}`?

Reply:
Sure. I went back and forth a few times and wound up picking "no" sort of at random.